classes.py

```diff
@@ -1 +1,34 @@
-# some_code
+class Rna(str):
+    sequence = ''
+    
+    def __init__(self, sequence):
+        if isinstance(self.sequence, str):
+            self.sequence = sequence.upper()
+        else:
+            raise TypeError
```

Comment:
You use only 4 necleotides in reverse_complement function. Can you add a check for the presence of extra letters in the line and give an error if so?

Reply:
Done